samples/artifacts/DataSummarization/NamedAggregation/NamedAggregation.siddhi

```diff
@@ -0,0 +1,96 @@
+@App:name("NamedAggregation")
+@App:description("Demonstrates how to calculate aggregates for various time granularity")
+
+/*
+Purpose:
+    This application demonstrates how to calculate aggregates for various time granularity.In this example the named aggregation gives the aggregation of events such as average of price and sum of price and top maximum price items with in last 5hours.
```

Comment:
last 5 hours is not a must, that is query specific. Improce it to say that 2 queries are demonstrated such as ...

Reply:
9ccf6a40df54b295ecf702859004e06cc81436b3 This commit fixed the above correction